### ISSUE TITLE
How to Enable/Disable bluetooth ?

### ISSUE THREAD

**Issue by Uefi1** [morrownr/8821cu#23]
The RTL8821CU IC provides both 2.5 / 5 GHz WLAN (802.11ac / 802.11abgn) and Bluetooth 4.2 in a single package.
Possible to switch Bluetooth ???

**Comment by morrownr** [comment]
Are you trying to switch Bluetooth support on or off?

**Comment by Uefi1** [comment]
YES

**Comment by morrownr** [comment]
That was not a question that can be answer with YES. Let me try again.

Do you want Bluetooth to be active?

Do you want Bluetooth to be inactive?

I cannot tell from your report whether Bluetooth is working and you want to turn it off or it isn't working and you want to turn it on.

What distro and version are you using?

**Comment by Uefi1** [comment]
How to turn it on or off Bluetooth what terminal commands ???

**Comment by morrownr** [comment]
I don't have an adapter that has the BT support so I have not played with it.

I have no idea how to answer your questions. I'll have to search when I have time.

Is the BT working right now?

What distro and version are you using?

**Comment by Uefi1** [comment]
It doesn’t work, I just don’t know how to switch

**Comment by morrownr** [comment]
One more time, what distro and version of Linux are you using?

You are not being very helpful. I cannot help if I don't understand the situation.

**Comment by Uefi1** [comment]
Porteus

**Comment by Digitalone1** [comment]
Bluetooth is not working to me either, but the chipset should support it.

**Comment by morrownr** [comment]
If anyone know how to do this, please post.

**Comment by t123yh** [comment]
> Does 8811cu support bluetooth or only 8821cu ???

Only 8821cu support bluetooth.


> It doesn’t work, I just don’t know how to switch

You have to have `CONFIG_BT_HCIBTUSB_RTL` option turned on when you compile linux kernel.

Also you must have `linux-firmware` package installed to obtain `rtl8821c_fw.bin` and `rtl8821c_config.bin`, both resides in `/usr/lib/firmware/rtl_bt` directory.

You can check with `dmesg` to find the following lines:

```
[    5.119156] Bluetooth: hci0: RTL: loading rtl_bt/rtl8821c_fw.bin
[    5.122775] Bluetooth: hci0: RTL: loading rtl_bt/rtl8821c_config.bin
```

Then you can enable bluetooth from userspace:

```
# systemctl enable bluetooth
# bluetoothctl
Agent registered
[CHG] Controller 54:C9:E0:00:xx:xx Pairable: yes
[bluetooth]# power on
Changing power on succeeded
[CHG] Controller 54:C9:E0:00:xx:xx Powered: yes
[bluetooth]# scan on
```
